packages/stockflux-watchlist/src/components/watchlist/Watchlist.js

```diff
@@ -191,17 +198,18 @@ const Watchlist = () => {
       className="watchlist"
       onDragStart={e => {
         onDragStart(e, watchlist, setDragOverIndex);
-        setDisplayPreview(true);
       }}
       onDragOver={e => {
         onDragOver(e, watchlist, dragOverIndex, setDragOverIndex);
       }}
       onDragEnd={() => {
         setDisplayPreview(false);
+        setDisplayDelete(false);
         resetDragState(setDragOverIndex);
       }}
       onDrop={e => {
         setDisplayPreview(false);
+        setDisplayDelete(false);
```

Comment:
And here there would only need to be one setDisplayX as false instead of two

Reply:
Changed to single variable method.